src/platformsh.js

```diff
@@ -0,0 +1,54 @@
+/**
+* Read number of CPUs from environment or fallback to the _private_ configuration property
+* Useful for determining the number of processes to fork.
+*/
+const num_of_cpus = () => {
+  try {
+    if(process.env['OMP_NUM_THREADS']) {
+      return process.env['OMP_NUM_THREADS'];
+    }
+    const config = require('/run/config.json');
```

Comment:
Can we read this file at the top-level, to make sure it's not blocking during e.g. an HTTP request?